include/seqan3/alphabet/quality/qualified.hpp

```diff
@@ -173,4 +173,134 @@ template <typename sequence_alphabet_type, typename quality_alphabet_type>
 qualified(sequence_alphabet_type &&, quality_alphabet_type &&)
     -> qualified<std::decay_t<sequence_alphabet_type>, std::decay_t<quality_alphabet_type>>;
 
+/*!\name Comparison operators for the qualified composition and it's inner alphabet type.
+ *\{
+ * \tparam alphabet_type The alphabet type of the qualified composition. Must
+ *                       model the seqan3::alphabet_concept.
+ * \tparam quality_type  The alphabet type of the qualified composition. Must
+ *                       model the seqan3::quality_type.
+ *
+ * This enables the comparison of a qualified composition value against a value
+ * of the inner alphabet type while ignoring it's quality score. The operators
+ * delegate the comparison operators of the alphabet type.
+ *
+ *```cpp
+ * qualified<dna4, phred42> q1{dna4::A, 7};
+ * qualified<dna4, phred42> q2{dna4::A, 30};
+ *
+ * std::cout << (q1 == dna4::A) << std::endl; // 1 (true)
+ * std::cout << (q1 == dna4::T) << std::endl; // 0 (false)
+ * std::cout << (q2 == dna4::A) << std::endl; // 1 (true)
+ * std::cout << (q2 == dna4::T) << std::endl; // 0 (false)
+ *```
+ *
+ */
+//!\brief Returns `true` if \p std::get<alphabet_type>(lhs) == \p rhs, `false` otherwise.
+template<alphabet_concept alphabet_type, quality_concept quality_type>
+constexpr bool operator==(qualified<alphabet_type, quality_type> const & lhs, alphabet_type const & rhs)
+{
+    return std::get<alphabet_type>(lhs) == rhs;
```

Comment:
Please don't use the type-based-std::get as potentially you could have two quality alphabets in the composition now and the get wouldn't know which one to take. Use the composition's member types instead.